src/rpc.server.ts

```diff
@@ -128,4 +149,18 @@ export class RPC implements IUsersServer {
         userRes.setHierarchyflat(Kartoffel.flattenHierarchy(user.hierarchy, user.job));
         return userRes;
     }
+
+    /**
+     * formatApproverInfo gets the Info object and returned it formatted.
+     * @param info is the user approver info from the phonebook
+     */
+    static formatApproverInfo(info: IApproverInfo): ApproverInfo {
+        const approverInfoRes: ApproverInfo = new ApproverInfo();
+
+        approverInfoRes.setCanapprove(info.isAdmin || info.canApprove);
```

Comment:
```suggestion
        approverInfoRes.setCanApprove(info.isAdmin || info.canApprove);
```

Reply:
cant change that, approverInfoRes is a class form the compiled proto file.